### PR TITLE
#11349 add required static disclaimer text

### PR DESCRIPTION
**Pull request by michael-caktus**
#### What's this PR do?
Implements [this basecamp todo item](https://3.basecamp.com/3120230/buckets/21497549/todos/4500174537). Specifically:
Adds some different text below the submit button on donation pages. This text will remain static even after we introduce a "new page template" that includes some small-print near this section.

#### How should this be manually tested?
View a donation page and observe the text. Notice how it changes when you change the amount, frequency, or payFees toggle.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No

### CODE REVIEW

File: spa/src/test-server.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { apiBaseUrl } from 'ajax/axios';
+
+const server = setupServer();
+
+server.listen({
+  // Log unhanlded requests for debugging.
```
Comment (benjaminEwhite) [inline spa/src/test-server.js:8]:
```suggestion
  // Log unhandled requests for debugging.
```

File: spa/src/test-utils.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import * as rtl from '@testing-library/react';
```
Comment (benjaminEwhite) [inline spa/src/test-utils.js:1]: Doesn't look like this is used anywhere, and unclear how related to the narrow scope of this PR.

**Review by benjaminEwhite — CHANGES_REQUESTED** [review]

File: spa/src/components/analytics/Analytics.spec.js [hunk 3]
```diff
@@ -1,52 +0,0 @@
-import React, { useEffect } from 'react';
```
[the remark this answers is not in the document]
Comment (benjaminEwhite) [reply spa/src/components/analytics/Analytics.spec.js:1]: @michael-caktus dug into this, and it looks like this is the only place we're testing the Facebook conversion tracking functionality. The other analytics are about page views specifically, and as you pointed out, are in fact tested by Cypress. This one is narrowly about testing FB conversion, and should probably stay.

**Comment by michael-caktus** [comment]
@benjaminEwhite there's a PR before this one that this branch is based from that includes all the test related stuff separately.

**Comment by benjaminEwhite** [comment]
@michael-caktus i had missed that PR. Just added comments about the analytics test (and a couple other small test-related things) on the other one.

File: spa/src/components/analytics/Analytics.spec.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (michael-caktus) [reply spa/src/components/analytics/Analytics.spec.js:1]: Thanks for looking in to that-- I've restored the test in the originating branch.

**Review by benjaminEwhite — APPROVED** [review]
Approved the test-related PR which addresses comments here.

**Pull request merged**